frame/staking/src/lib.rs

```diff
@@ -761,6 +766,105 @@ impl<AccountId, Balance: HasCompact + Zero> UnappliedSlash<AccountId, Balance> {
 	}
 }
 
+/// Something that defines the maximum number of nominations per nominator based on a curve.
+pub trait NominationsQuota<Balance> {
+	/// Maximum number of nominations. The method `curve` implements the nomination quota curve and
```

Comment:
This doc seems to describe the whole trait more so than this type item.

This type items should be documented as "what is typically used with BoundedVec" etc etc.